dist/givey.js

```diff
@@ -2128,20 +2139,36 @@ String.prototype.pluralize = function () {
           var model = new klass.models[type](data[type]);
           resolve(model);
         }
-      }, function (error) {
-        if (window.console && window.console.error) {
-          console.error('GIVEY DATA ERROR:', error);
-        } else {
-          alert('GIVEY DATA ERROR: ' + error);
-        }
-      });
+      };
+      app.getJSON(url, {}, success_callback, failure_callback);
```

Comment:
This is not using `jQuery.getJSON`, this is `app.getJSON` which is a custom wrapper around that method. The wrapper is responsible for adding the access_token to each request.